### PR TITLE
PVP 191

### PR DESCRIPTION
**Pull request by youreondrugs**

### CODE REVIEW

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 1]
```diff
@@ -302,7 +305,7 @@ fun Day(
             }
         }
 
-        if (expand) {
+        /*if (expand) {
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:308]: Why is it commented out?

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 2]
```diff
@@ -658,8 +661,8 @@ fun TaskFilterBar(
 ) {
     // Fixed to take up the whole screen for now as it bugs out in Weekly view,
     // replace Modifier.width with Modifier.weigh(1f)
-    val screenWidth = LocalConfiguration.current.screenWidthDp.dp
-    val chipWidth = screenWidth / TaskFilter.values().size
+    //val screenWidth = LocalConfiguration.current.screenWidthDp.dp
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:664]: Why are these commented out?

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 3]
```diff
@@ -716,7 +721,7 @@ fun DayPage(
     tasks: List<Task>,
     page: Int
 ) {
-    val tasksForDay = tasks.filter { task -> task.scheduledAt.toLocalDate() == date }
+    //val tasksForDay = tasks.filter { task -> task.scheduledAt.toLocalDate() == date }
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:724]: Why is this commented? If not required, just remove.

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 4]
```diff
@@ -754,17 +762,93 @@ fun Week(
...
+    val filteredTasks = filterTasks(tasks, selectedFilter)
+    val screenHeight = LocalConfiguration.current.screenHeightDp.dp
+
+    Column {
+        HorizontalPager(
+            state = pagerState,
+            modifier = modifier
+                .padding(horizontal = 16.dp)
+                .height(screenHeight / 3),
+            contentPadding = PaddingValues(70.dp, 0.dp),
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:775]: Split params.

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 5]
```diff
@@ -754,17 +762,93 @@ fun Week(
         pageCount = { days.size }
     )
     val currentPage = pagerState.currentPage
+    var selectedFilter by remember { mutableStateOf(TaskFilter.Daily) }
+    val filteredTasks = filterTasks(tasks, selectedFilter)
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:766]: Split params. Then this block of code will be of multiple lines, separate it by a single line (top/bot).

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 6]
```diff
@@ -754,17 +762,93 @@ fun Week(
...
+    Column {
+        HorizontalPager(
+            state = pagerState,
+            modifier = modifier
+                .padding(horizontal = 16.dp)
+                .height(screenHeight / 3),
+            contentPadding = PaddingValues(70.dp, 0.dp),
+        ) { page ->
+            val tasksForDay = tasks.filter { task -> task.scheduledAt.toLocalDate() == dates[page] }
+            DayPage(
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:778]: Should separate.

**Comment by vytjar** [comment]
The weekly screen now looks like this:

image

- Can't select task type, text "No daily task selected" overlaps with the bar.
- Pressing day does not open task creation, like in development
- Also, no way to see new daily view that Lukas implemented:

image

**Review by klaidoshka — APPROVED** [review]

**Review by vytjar — APPROVED** [review]

**Pull request merged**